src/client/components/login-form.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import { Route, Redirect } from "react-router-dom";
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      isLoggedin: false
+    };
```

Comment:
I really like using class fields for this:
```suggestion
state = {
  ...
}
```
No need to make a constructor, then.

Reply:
removed constructor.